autodoc-admin-api/data.lua

```diff
@@ -1473,4 +1434,136 @@ return {
     }
   },
 
+--------------------------------------------------------------------------------
+-- Overrides for DB-less mode
+--------------------------------------------------------------------------------
+
+  dbless = {
+
+    intro = [[
+
+    <div class="alert alert-info.blue" role="alert">
+      This page refers to the Admin API for running Kong configured without a
+      database, managing in-memory entities via declarative config.
+      For using the Admin API for Kong with a database, please refer to the
+      <a href="/{{page.kong_version}}/admin-api">Admin API for Database Mode</a> page.
+    </div>
+
+    Kong comes with an **internal** RESTful Admin API for administration purposes.
+    In [DB-less mode][db-less], this Admin API can be used to load a new declarative
+    configuration, and for inspecting the current configuration. In DB-less mode,
+    the Admin API for each node functions independently, reflecting the memory state
```

Comment:
nitpick / ignore if the convention is otherwise
```suggestion
    the Admin API for each Kone node functions independently, reflecting the memory state
```